src/makeCalculator.js

```diff
@@ -37,7 +37,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const resultObj = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(method, number) {
+      this.result = method(number, this.result);
+
+      return this;
+    },
+
+    add(number, result) {
+      return number + result;
+    },
+    subtract(number, result) {
```

Comment:
```suggestion
    add(number, result) {
      return number + result;
    },
    
    subtract(number, result) {
```

Add empty line between methods